packages/components/src/CollapsiblePanel/CollapsiblePanel.scss

```diff
@@ -32,95 +35,83 @@ $tc-collapsible-panel-btn-color: #555964;
 	}
 
 	&.col-6 {
-		@include type-col(calc(100%/6));
+		@include type-col(calc(100% / 6));
 	}
 
 	&.col-7 {
-		@include type-col(calc(100%/7));
+		@include type-col(calc(100% / 7));
 	}
 
 	&.col-8 {
-		@include type-col(calc(100%/8));
+		@include type-col(calc(100% / 8));
 	}
 
 	&.col-9 {
-		@include type-col(calc(100%/9));
+		@include type-col(calc(100% / 9));
 	}
 
 	&.col-10 {
 		@include type-col(calc(10%));
 	}
 }
-@mixin header-style {
-	display: flex;
-	align-items: center;
-	justify-content: space-between;
-	padding-right: $tc-collapsible-panel-padding-larger;
-
-	> * {
-		@include col-style;
-	}
-}
 
-.group {
-	justify-content: space-between;
-
-	> * {
-		margin: 0 $tc-collapsible-panel-padding-normal;
-	}
-
-	:global(.label) {
-		white-space: nowrap;
-		text-overflow: ellipsis;
-		overflow: hidden;
-		font-size: 1em;
-	}
-}
-
-.tc-panel {
-	:global(.panel-heading) {
-		@include header-style;
-		padding-right: $tc-collapsible-panel-padding-larger + $tc-collapsible-panel-padding-normal;
-	}
+// due to react-bootstrap
+:global(.panel .panel) {
+	margin: 0;
+	border: none;
 }
 
 .tc-collapsible-panel {
 	position: relative;
 
-	.caret {
-		position: absolute;
-		right: 0;
-		top: 0;
-		bottom: 0;
+	&.selected {
+		border-left: 4px solid $tc-collapsible-selected-panel-border-color;
 	}
 
-	:global(.panel-heading) {
-		h4 {
-			line-height: $line-height-base;
+	&.open {
+		.toggle svg {
+			transform-origin: center;
```

Comment:
As already discussed, that property should not be when it's toggled, you should set it for `svg` rather than `.toggle svg`